az-digital/az_quickstart
# Add Cards Paragraph

This PR adds a cards paragraph and behavior settings based on @joeparsons `composite-fields` branch.

## Description
This PR adds a cards paragraph and behavior settings based on @joeparsons `composite-fields` branch.

## Related Issue
#289 

## How Has This Been Tested?

- Create page content
- Attempt to add cards paragraph content, and add several cards within the paragraph
- Save content and verify the cards are rendered correctly
- Edit content and determine if the column setting options on the cards paragraphs work correctly.

## Types of changes


## Checklist:
